CHANGELOG.md

```diff
@@ -95,6 +95,8 @@ for further information about branching and tagging conventions.
 - Receivable distribution chart (Loyalty)
 - Receivable-payable delta chart (Loyalty)
 - Account number field to the invoice form (Loyalty)
+- "Copy" button to EmailGetter
+- Fix for handling a "400" errors
```

Comment:
actually, in the code, you fixed handling of a TransactionError, despite js-sdk returns this kind of error in cases of 400 error code you should not describe the error code in the changelog, because it was encapsulated.